File: C#/백준/Bronze/6778. Which Alien？/Which Alien？.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            int b = int.Parse(Console.ReadLine());
            if(a>=3 && b<=4){
                Console.WriteLine("TroyMartian");
            }
            if(a<=6 && b>=2){
                Console.WriteLine("VladSaturnian");
            }
            if(a<=2 && b<=3){
                Console.WriteLine("GraemeMercurian");
            }
        }
    }
}

File: Java/백준/Bronze/6778. Which Alien？/Which Alien？.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt(); int b = in.nextInt();
        if(a>=3 && b<=4){
            System.out.println("TroyMartian");
        }
        if(a<=6 && b>=2){
            System.out.println("VladSaturnian");
        }
        if(a<=2 && b<=3){
            System.out.println("GraemeMercurian");
        }
    }
}